code/modules/mob/living/carbon/human/species/abductor.dm

```diff
@@ -16,7 +15,7 @@
 		)
 
 	species_traits = list(NO_BLOOD, NO_BREATHE, VIRUSIMMUNE, NOGUNS, NO_EXAMINE)
-
+	dies_at_threshold = TRUE
```

Comment:
I understand the change for other species, but why remove the abductor's heart and not have them use this new system? There's no difference medically between an abductor and a human as far as I know. Is this for balance reasons?

Reply:
They also have `NO_BLOOD`, which means the heart really has no reason to be there. It doesn't do anything; the flag prevents having a heart attack, which is a major component of the new crit system. They also have `NO_BREATHE` which is also problematic.